tools/prepare_lava_job.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+"""
+This script is designed to generate a LAVA Job-file in yaml, which
+can be used with our RevPi devices in order to flash and perform tests
+with LAVA automatically, thus saving time finding errors in the images.
+"""
+
+import argparse
+import os
+import pkg_resources
+import subprocess
+import sys
+import yaml
+
+parser = argparse.ArgumentParser(
+    description=__doc__,
+    epilog="Default behaviour: Generate a LAVA Job to flash and run tests on a given RevPi device.",
+)
+
+parser.add_argument(
+    "-d",
+    "--device-type",
+    dest="device_type",
+    nargs="?",
+    help="name of device type to submit job. Default: RevPi_Core",
```

Comment:
```suggestion
    help="Device type to use for testing",
```

The default is given by the used template. This can change in the future.

A ticket should be created for checking that all the values are given, either as arguments to the script or from the template.